cityehr-quick-start-guide/src/main/xslt/create-topic-html.xslt

```diff
@@ -130,7 +130,7 @@
   <xsl:function name="htop:petal-edit-url" as="xs:string">
     <xsl:param name="topic" as="element(topic)" required="yes"/>
     <xsl:variable name="petal-source-file-uri" select="com:document-uri($topic)" />
-    <xsl:variable name="petal-source-file" select="substring-after($petal-source-file-uri, concat($petal-github-repo-name, '/'))" />
+    <xsl:variable name="petal-source-file" select="substring-after($petal-source-file-uri, concat($petal-github-repo-name,  '/', $petal-github-repo-name,  '/'))" />
```

Comment:
@marmoure This expression `concat($petal-github-repo-name, '/', $petal-github-repo-name, '/')` doesn't make sense, and `petal-source-file` will be empty.